src/containers/offer-details/offer-carousel/OfferCarousel.tsx

```diff
@@ -55,9 +55,12 @@ const OfferCarousel = ({ offer }: OfferCarouselProps) => {
     [offer]
   )
 
-  const { response } = useAxios<GetOffersResponse>({
-    service: getOffers,
-    defaultResponse
+  const { data: response } = useQuery<GetOffersResponse>({
```

Comment:
Please, use `ItemsWithCount` generic type for consistent type usage across the codebase
```suggestion
  const { data: response } = useQuery<ItemsWithCount<Offer>>({
```

Reply:
Fixed.